pkg/common/utils/gang_utils.go

```diff
@@ -32,6 +32,11 @@ import (
 	"github.com/apache/yunikorn-k8shim/pkg/log"
 )
 
+const (
+	k8sDomain       = "kubernetes.io"
+	hugepagesPrefix = "hugepages-"
```

Comment:
Nit: these can be referenced by (core/v1)  `v1.ResourceHugePagesPrefix`

Reply:
done